core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackController.java

```diff
@@ -521,13 +521,14 @@ public void onServiceQueried()  {}
      * Should be used by classes which implement the OnSeekBarChanged interface.
      */
     public float onSeekBarProgressChanged(SeekBar seekBar, int progress,
-                                          boolean fromUser, TextView txtvPosition) {
+                                          boolean fromUser, TextView txtvPosition, TextView seekDisplay) {
         if (fromUser && playbackService != null && media != null) {
             float prog = progress / ((float) seekBar.getMax());
             int duration = media.getDuration();
             TimeSpeedConverter converter = new TimeSpeedConverter(playbackService.getCurrentPlaybackSpeed());
             int position = converter.convert((int) (prog * duration));
             txtvPosition.setText(Converter.getDurationStringLong(position));
+            seekDisplay.setText(Converter.getDurationStringLong(position));
```

Comment:
I am wondering if we still need to update the position TextView in the line above, now. When seeking, it might be interesting to see where you started from.